button/popup.css

```diff
@@ -282,8 +286,8 @@ http://matthewlein.com/ceaser/
 
 .removenote2 {
    transition: 0.3s cubic-bezier(0.075, 0.820, 0.165, 1.000); /* easeOutCirc */;
-   height: 0;
-   padding: 0;
+   height: 0px;
+   padding: 0px;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/2651364002/issues/source?bid=2884821&fileBranchId=3324709#l290)